config/locales/sufia.en.yml

```diff
@@ -204,6 +204,8 @@ en:
         language: "The language of the file content."
         publisher: "The person or group making the file available. Generally this is the institution."
         rights: "Licensing and distribution information governing access to the file. Select from the provided drop-down list. &lt;em&gt;This is a required field&lt;/em&gt;."
+      collection:
+        creator_html: "The person or group responsible for the collection. Usually this is the author of the collected content. Personal names should be entered with the last name first, e.g. &quot;Smith, John.&quot; &lt;em&gt;This is a required field&lt;/em&gt;."
```

Comment:
@yinlinchen what @awead  is saying is you can put in "Smith, John" instead of &quot;Smith, John.&quot;  ect.